src/main/java/com/commandgeek/GeekSMP/listeners/discord/DiscordMessageCreateListener.java

```diff
@@ -89,7 +89,7 @@ public static boolean command(Message message) {
                     MuteManager.mute(player.getName(), null, null, sender);
                     new MessageManager("discord-mute-permanent")
                             .replace("%user%", user.getMentionTag())
-                            .replace("%player%", player.getName())
+                            .replace("%player%", player.getName(), true)
```

Comment:
Adding "true" to these placed doesn't make sense to me, and it is throwing errors. Is there a specific reason for this?

Reply:
Checks passed, it also didnt throw any errors for me...
I overloaded the replace method to have a third argument which escapes the markdown of the replacment text and then replaces it

Reply:
You're right, my bad. I didn't notice the additional replace method you created.